main.py

```diff
@@ -36,11 +36,20 @@
         default=False,
         help="Collect patches but don't apply them",
     )
+    parser.add_argument(
+        "--match_aliquot",
+        action="store_true",
+        default=False,
+        help=(
+            "Match dbGaP `SAMPLE_ID` to `external_aliquot_id` in the dataservice.\n"  # noqa E501
+            " - Defaults to match on `external_sample_id`"
+        ),
+    )
     args = parser.parse_args()
     print(f"Args: {args.__dict__}")
 
     patches, alerts = ConsentProcessor(
-        args.server, args.db_url
+        args.server, args.db_url, args.match_aliquot
```

Comment:
I think this argument should go into get_patches_for_study since it seems like a study-specific issue?